content_scripts/selector.js

```diff
@@ -31,7 +31,7 @@ const SEL = {
     strategyReportHeader: '#bottom-area div.backtesting div[class^="reportContainer"] table thead > tr > th',
     strategyReportRow: '#bottom-area div.backtesting div[class^="reportContainer"] table tbody > tr',
     strategyListOptions: 'div[role="listbox"] div[data-name="menu-inner"] div[role="option"] span[class^="label-"]',
-    strategyDefaultElement: '#study-defaults-manager',
+    strategyDefaultElement: 'div[data-name="indicator-properties-dialog"] div[class^="footer-"] div[class^="buttons-"]',
```

Comment:
why not `#property-actions`?

Reply:
i didn't see it, i agree with this id instead of classes